lib/matplotlib/ticker.py

```diff
@@ -1159,11 +1144,41 @@ def _non_decade_format(self, sign_string, base, fx, usetex):
         if is_close_to_int(coeff):
             coeff = round(coeff)
         if usetex:
-            return (r'$%s%g\times%s^{%d}$') % \
-                                        (sign_string, coeff, base, exponent)
+            return ((r'$%s%g\times%s^{%d}$') %
+                    (sign_string, coeff, base, exponent))
         else:
             return ('$%s$' % _mathdefault(r'%s%g\times%s^{%d}' %
-                                        (sign_string, coeff, base, exponent)))
+                    (sign_string, coeff, base, exponent)))
+
+
+class SymmetricalLogFormatter(LogFormatterSciNotation):
+    def __init__(self, linthresh, base=10.0, labelOnlyBase=False,
+                 minor_thresholds=None):
+        self._linthresh = linthresh
+        super().__init__(base, labelOnlyBase, minor_thresholds)
+
+    def __call__(self, x, pos=None):
+        usetex = rcParams['text.usetex']
+        if np.abs(x) <= self._linthresh:
+            if usetex:
+                return f'${x}$'
+            else:
+                return '$%s$' % _mathdefault(f'{x}')
```

Comment:
I like f-strings very very much but perhaps `f'{x}'` is a bit overdoing it compared to `str(x)`...